format.md

```diff
@@ -0,0 +1,33 @@
+---
+---
+
+# The format of a How We Have Been Saving The Planet This Week newsletter
+
+We write a weekly newsletter on awesome things. This is mainly these things:
+- About how we are improving our environment creating a more liveable world for humans.
```

Comment:
We write about, and share articles that are about how we are improving our environment creating a more liveable world for humans.